<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AntScopeUpdateDialog</class>
 <widget class="QDialog" name="AntScopeUpdateDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>141</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QProgressBar" name="downloadProgressBar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>681</width>
     <height>31</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="installUpdates">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>10</y>
     <width>501</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Do you want to install new version?</string>
   </property>
  </widget>
  <widget class="QPushButton" name="dowloadNow">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>100</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Download and Install</string>
   </property>
  </widget>
  <widget class="QPushButton" name="close">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>100</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QLabel" name="newVersion">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>10</y>
     <width>441</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>New version of AntScope 2 is available!</string>
   </property>
  </widget>
  <widget class="QPushButton" name="afterClosing">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>100</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Download and Install after closing</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
